### ISSUE TITLE
Followup on StyledInputTags accessibility

### ISSUE THREAD

**Issue by kewitz** [opencollective/opencollective#3078]
The new popup StyledInputTags is lacking some accessibility features, like being able to add/remove tags using only the keyboard.

Make sure that the user can:
- [ ] Easily add tags with keyboard
- [ ] Use suggestions with keyboard
- [ ] Remove existing tags with keyboard

**Comment by Betree** [comment]
:gift: :money_with_wings: A **$200** bounty was attached to this issue. Anyone submitting a Pull Request will be rewarded with $200 when the Pull Request is reviewed, accepted and merged.

:arrow_right: [More info](https://docs.opencollective.com/help/developers/bounties)

**Comment by brymut** [comment]
Hi @JoyLubega may I please work on this while you work on the other bounty you're assigned to?

**Comment by JoyLubega** [comment]
its okay. @brymut you can take this and i stay with the other one

**Comment by brymut** [comment]
Thank you @JoyLubega

**Comment by Betree** [comment]
:trophy: This issue has been completed by @brymut  

To claim the bounty, you can either:
- Submit an invoice to https://opencollective.com/engineering/expenses/new (please mention the issue number)
- Or ask for an Open Collective gift card of the same value (to donate to other collectives)
